web/components/admin/FcAdminPermissions.vue

```diff
@@ -18,7 +18,7 @@
             sort-by="UNIQUE_NAME"
             :sort-desc="false"
             :sort-keys="sortKeys">
-            <template v-slot:item.UNIQUE_NAME="{ item }">
+            <template v-slot:[`item.UNIQUE_NAME`]="{ item }">
```

Comment:
This was a linter complaint. Has nothing to do with this PR, but I needed to fix it to commit. 🤷